pkgs/development/libraries/libbladeRF/default.nix

```diff
@@ -35,6 +35,6 @@ stdenv.mkDerivation rec {
     description = "Supporting library of the BladeRF SDR opensource hardware";
     license = licenses.lgpl21;
     maintainers = with maintainers; [ funfunctor ];
-    platforms = platforms.linux;
+    platforms = with platforms; linux;
```

Comment:
It might work on darwin, if `udev` was made optional.
 However we should probably let somebody test it on the platform first.

Reply:
According to their web page this package should support Darwin but I have to possibility to test it.
I'll give it a try and make udev optional.